src/input-group.scss

```diff
@@ -201,6 +201,31 @@ $fd-input-border-radius: var(--sapField_BorderCornerRadius);
 
     border: none;
     width: 100%;
+
```

Comment:
Please update component structure description (Same for the rest of edited components)
```
/*!
.fd-input-group+(--inline)
    .fd-input-group__addon+(--button)
*/
```

Reply:
Done